disvae/losses.py

```diff
@@ -325,30 +332,29 @@ def _kl_normal_loss(mean, logvar, storer=None):
     return total_kl
 
 
-def _permute_dims(latent_dist):
+def _permute_dims(latent_sample):
```

Comment:
how are the results compared to before?

Reply:
I'm currently training on the server (300 epochs), I'll post the results in slack when finished (+- couple hours)